src/components/link/_link.scss

```diff
@@ -89,9 +85,13 @@
     }
 
     &:visited {
-      color: $interactive-01;
+      color: $link-visited;
     }
   }
+
+  .#{$prefix}--disabled {
+    color: $disabled-02;
```

Comment:
I think we can also add `pointer-events: none` to prevent clicking on a disabled link